src/main/java/org/sagebionetworks/bridge/models/assessments/config/ConfigVisitor.java

```diff
@@ -20,7 +20,7 @@
     private Errors errors;
     private Map<String, Validator> validators;
     
-    ConfigVisitor(Map<String, Validator> validators, Errors errors) {
+    public ConfigVisitor(Map<String, Validator> validators, Errors errors) {
         this.errors = errors;
         this.validators = (validators == null) ? ImmutableMap.of() : validators;
     }
```

Comment:
If this was made public to enable testing, can you move the visitor to the validation package as well? It's part of the AssessmentConfigValidator.

Reply:
Yah, I changed it to public when I moved AssessmentConfigValidator. I'll move this into the validators package too and undo the change to public.